content/working_with/console/widgets/deployments.md

```diff
@@ -103,12 +83,47 @@ For detaching the current site, leave the `Site name` input empty and toggle the
 ![Set Site]( /images/ui/widgets/blueprint-deployments_set-site.png )
 
 
+#### Managing Labels
+
+1. Click **Manage Labels** in the action menu.
+2. Add/Remove labels to/from the deployment. 
+   NOTE: New labels (not existing in the system) are marked with blue color.
+3. Click **Apply**.
+
+![Manage Labels]( /images/ui/widgets/blueprint-deployments_manage-labels.png )
+
+You can learn more about labels [here]({{< relref "cli/orch_cli/deployments.md#labels" >}}).
+
+
 #### Deleting a Deployment
 
 1. Click **Delete** or **Force Delete** in the action menu.
 2. When prompted for deployment removal confirmation, click **Yes**.
 
 
+### Executing a workflow
+
+1. Click the cogs icon (![cogs icon]( /images/ui/icons/execute-workflow-icon.png ))  
+   and select the workflow you want to execute.
+2. Provide values for workflow parameters.
+3. Click **Execute**.
+
+![Execute workflows menu]( /images/ui/widgets/blueprint-deployments_workflows-menu.png )
+
+For **Install** or **Uninstall** workflows you will also be able to track the progress of the execution as
+at the bottom of the deployment row there will be thin line visible. Progress is calculated based on number of node instances installed (in case of install workflow) or deleted (in case of uninstall workflow).
```

Comment:
```suggestion
at the bottom of the deployment row, there will be a thin line visible. Progress is calculated based on the number of node instances installed (in case of an install workflow) or deleted (in case of an uninstall workflow).
```